_tests/tool/travis.sh

```diff
@@ -10,6 +10,8 @@ set -e
 dartanalyzer --fatal-warnings .
 
 pub run test -p vm -x codegen
+
+export BUILD_MAX_WORKERS_PER_TASK=1
```

Comment:
Will this work if we just define it at the top-level `travis.sh`? 

That way, it would apply to all of the sub-packages that use `build_runner`.

Reply:
oh it should, I will try adding that